launcher.js

```diff
@@ -0,0 +1,45 @@
+// A "launcher" to start the (Node version of the) router from the command line.
```

Comment:
If this is the primary way to start the application, why do we call the file <code>launcher.js</code>? Should this not be the <code>index.js</code> file? Or are there reasons for this separation? If so, should we at least ensure the <code>"main"</code> property in <code>package.json</code> points to this file, so that people can start it up with a simple <code>node .</code>? Or do we want a shell script wrapper that will do so?